package.json

```diff
@@ -7,189 +7,101 @@
+        "customize-cra": "^0.8.0",
         "d3": "^5.7.0",
-        "date-fns": "^2.0.0-alpha.27",
-        "dotenv": "4.0.0",
-        "dotenv-expand": "4.2.0",
         "es6-tween": "^5.4.3",
-        "extract-text-webpack-plugin": "3.0.2",
-        "file-loader": "0.11.2",
-        "fork-ts-checker-webpack-plugin": "^0.5.2",
-        "fs-extra": "3.0.1",
         "gh-pages": "^2.1.1",
-        "html-webpack-plugin": "2.29.0",
-        "jest": "22.4.2",
         "jsbi": "^2.0.5",
         "json-bigint": "^0.3.0",
         "lodash": "^4.17.10",
         "long": "^4.0.0",
-        "mobx": "^5.1.0",
-        "mobx-react": "^5.2.5",
-        "mobx-react-lite": "^1.0.1",
+        "mobx": "^5.15.0",
+        "mobx-react": "^6.1.4",
+        "mobx-react-lite": "^1.5.0",
         "nanoid": "^2.0.1",
-        "node-sass": "^4.9.3",
+        "node-sass": "^4.13.0",
         "normalize.css": "^8.0.0",
-        "object-assign": "4.1.1",
         "particulate": "^0.3.3",
-        "pixi.js": "^4.8.1",
-        "postcss-flexbugs-fixes": "3.2.0",
-        "postcss-loader": "2.0.8",
-        "promise": "8.0.1",
+        "pixi.js": "^4.8.0",
         "qrcode": "^1.3.3",
         "qrcode.react": "^0.9.3",
-        "raf": "3.4.0",
-        "react": "16.8.3",
+        "react": "^16.11.0",
+        "react-app-rewired": "^2.1.5",
         "react-codemirror2": "^5.1.0",
         "react-confirm-alert": "^2.3.0",
-        "react-dev-utils": "^5.0.1",
-        "react-dom": "16.8.3",
+        "react-dom": "^16.11.0",
         "react-dropdown": "^1.6.4",
         "react-dropzone": "^10.1.8",
         "react-infinite-scroller": "^1.2.4",
         "react-notifications-component": "^1.1.1",
         "react-qr-code": "^0.1.3",
         "react-qr-svg": "^2.2.1",
-        "react-router-dom": "^4.3.1",
+        "react-router-dom": "^5.1.2",
+        "react-scripts": "3.2.0",
         "react-sizeme": "^2.5.2",
         "react-table": "^6.9.2",
         "react-transition-group": "^2.8.0",
-        "reconnecting-websocket": "^4.1.10",
-        "resolve": "1.6.0",
-        "sass-loader": "^7.1.0",
+        "reconnecting-websocket": "^4.2.0",
         "smart-buffer": "^4.0.2",
-        "source-map-loader": "^0.2.1",
         "store": "^2.0.12",
-        "style-loader": "0.19.0",
         "styled-components": "^4.1.3",
-        "sw-precache-webpack-plugin": "0.11.4",
         "three": "^0.103.0",
         "three-trackballcontrols": "github:claudiucelfilip/three-trackballcontrols",
-        "ts-jest": "22.0.1",
-        "ts-loader": "^5.3.3",
-        "tsconfig-paths-webpack-plugin": "^2.0.0",
         "tslint": "^5.7.0",
         "tslint-config-prettier": "^1.10.0",
         "tslint-react": "^3.2.0",
         "tweetnacl": "^1.0.0",
-        "uglifyjs-webpack-plugin": "^1.1.8",
-        "url-loader": "0.6.2",
-        "vis": "^4.21.0",
+        "typescript": "3.7.2",
         "wabt": "^1.0.10",
-        "wavelet-client": "^1.1.0",
+        "wavelet-client": "^2.0.0",
```

Comment:
Can we test with the rc published version

Reply:
Changed with rc2